source/server/listener_manager_impl.h

```diff
@@ -195,6 +204,8 @@ class ListenerManagerImpl : public ListenerManager, Logger::Loggable<Logger::Id:
   std::list<DrainingListener> draining_listeners_;
   std::list<WorkerPtr> workers_;
   bool workers_started_{};
+  bool inbound_listeners_stopped_{};
```

Comment:
switch these 2 bools to a "stop state" enum?

Reply:
Related to my other comment above. I can change based on what we decide there